express-notes-api/get.js

```diff
@@ -0,0 +1,29 @@
+function getId(app, data) {
+  app.get('/api/notes/:id', (req, res) => {
+    // Is this fine to use as well? const id = req.url.split('/')[3];
```

Comment:
It can be used in this case, but I wouldn't recommend it. It can be hard to parse that `url` correctly and express does it for you, so you should use what they do. But in this case, the two are identical.